src/cryptsetup/cryptsetup.c

```diff
@@ -950,6 +955,15 @@ static int run(int argc, char *argv[]) {
                         }
                 }
 
+                /* since cryptsetup 2.3.0 (Feb 2020) */
+                #ifdef CRYPT_BITLK
+                if (!arg_type || STR_IN_SET(arg_type, ANY_LUKS, CRYPT_BITLK)) {
+                        r = crypt_load(cd, CRYPT_BITLK, NULL);
+                        if (r < 0)
+                                return log_error_errno(r, "Failed to load Bitlocker superblock on device %s: %m", crypt_get_device_name(cd));
+                }
+                #endif
+
```

Comment:
hmm, so you are reusing attach_luks_or_plain() to do the actual activation of the volume. That function should be renamed then, to attach_luks_or_plain_or_bitlk() or so